README.md

```diff
@@ -14,36 +14,37 @@ Operator to manage New Relic resources.
 
 Currently enables management of Alert Policies and NRQL Alert Conditions.
 
-# Quick start test drive from zero, running kubernetes in a docker container locally with kind
+# Running kubernetes in a docker container locally with kind
 
-Get docker, kubectl, kustomize and kind installed
-``` bash
-brew cask install docker
-brew install kustomize kubernetes-cli kind
-```
+1. Get docker, kubectl, kustomize and kind installed
 
-Create a test cluster with kind
+   ``` bash
+   brew cask install docker
+   brew install kustomize kubernetes-cli kind
+   ```
 
-``` bash
-kind create cluster --name newrelic
-kubectl cluster-info
-```
+1. Create a test cluster with kind
 
-Install cert-manager
+   ``` bash
+   kind create cluster --name newrelic
+   kubectl cluster-info
+   ```
 
-``` bash
-kubectl apply --validate=false -f https://github.com/jetstack/cert-manager/releases/download/v0.15.0/cert-manager.yaml
-```
+1. Install cert-manager
 
-Note: This takes a minute or two to finish so wait a minute before going on to the next step. 
-You can also confirm it's running with the command `kubectl rollout status deployment -n cert-manager cert-manager-webhook`
+   ``` bash
+   kubectl apply --validate=false -f https://github.com/jetstack/cert-manager/releases/download/v0.15.0/cert-manager.yaml
+   ```
 
-Install the operator in the test cluster.
+   > **Note:** This takes a minute or two to finish so wait a minute before going on to the next step.
 
-``` bash
-kustomize build github.com/newrelic/newrelic-kubernetes-operator/config/default/ \
-  | kubectl apply -f -
-```
+   You can also confirm it's running with the command `kubectl rollout status deployment -n cert-manager cert-manager-webhook`
+
+1. Install the operator in the test cluster.
+
+   ``` bash
+   kustomize build ./configs/default | kubectl apply -f -
```

Comment:
so we specifically had the example pointing directly to the direct github link because we wanted users to be able to install the operator without needing to clone the repo or download anything at all

Reply:
Seems like a fair point.

Reply:
Ah okay, that's easy enough to change back and update the section to reflect remote install vs local.